src/User/UserContainer.jsx

```diff
@@ -14,27 +15,44 @@ const UserContainer = ({ token }) => {
     setIsLoading(true);
 
     try {
-      const user = await getUserById(userId, token);
+      const user = await getUserById(userId, sessionToken.token);
       setUser(user);
+      setUpdatedUser({
```

Comment:
Would you be able to explain what this line and the one above it are doing? I'm a little confused

Reply:
I think after reading it a bit more, you should consider moving the updatedUser state into the UpdateProfile component